lib/sinatra/base.rb

```diff
@@ -682,6 +682,10 @@ def erb(template, options = {}, locals = {}, &block)
       render(:erb, template, options, locals, &block)
     end
 
+    def erubi(template, options = {}, locals = {}, &block)
+      render(:erubi, template, options, locals, &block)
+    end
```

Comment:
I think this isn't necessary. Actually the `erubis` helper is deprecated.
What do you think?

Reply:
I thought that it was better to align with erubis and added helper, but it is certainly not necessary. I'll remove it.

Reply:
I fixed it

Reply:
Did you see https://github.com/sinatra/sinatra/commit/3e641dbf667a8a717b209ec70eb938ba3962a27c ?
New helper seems not reasonable because of above commit.
Thanks for fixing that.